GSA/digitalgov.gov
# Feature - Merge of topics for cleanup

## Summary

- This PR will handle the section of merging topics. 
- Old URLs were added to Wayback Machine 
- Total topics are now at 440

Spreadsheet for topics that are being merged is listed in the ticket named `DG - Topics: Remove ROT topics`

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/feature/cm-merge-topics/)



### Solution
Removed old slug and replaced any ones listed with the "new-merged" slug, and renamed ones as needed.

### How To Test

1. Review spreadsheet and verify that URLs are no longer available 
2. Check that all listed URLs are captured within this PR 

---

### Dev Checklist


Tagging Supervisor - @rayestrada
